recipes/elyra/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "elyra" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/0b/0d/7355f28392a40bca18a373c2f6c55306a3f345ea257afa6e3a074a6e9785/elyra-{{ version }}-py3-none-any.whl
+  sha256: 3c713cc33cb309caaa41176a33e187012b5092ec611b1ad063e9ea65a743806b
```

you will need to change it as well
```suggestion
  sha256: fb032f94de2fbc8e183d3dfa4abeeb4692f3a1c2daa7fa9559a3308842df44a0
```